graphics/unpaper/Portfile

```diff
@@ -2,43 +2,40 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           meson 1.0
 
-# When updating to the next version, please switch to xz tarball
-# and no longer autoreconf.
```

Comment:
Is there a reason you can't do what this comment says?

Reply:
beyond using the xz file from the github release, I'm not sure what you mean? The build system was changed upstream.

Reply:
Yes, I meant using the xz release tarball instead of the github auto-generated one. Thanks.

Reply:
got it. fixed.